src/Arcus.Templates.Tests.Integration/WebApi/WebApiProject.cs

```diff
@@ -191,36 +191,13 @@ public static WebApiProject CreateNew(TestConfig configuration, WebApiProjectOpt
             Uri baseUrl = configuration.GenerateRandomLocalhostUrl();
             var project = new WebApiProject(baseUrl, configuration, templateDirectory, fixtureDirectory, outputWriter);
             project.CreateNewProject(projectOptions);
-            project.UpdateFileInProject("Program.cs", contents => project.RemovesUserErrorsFromContents(contents));
-
-            return project;
-        }
-
-        /// <summary>
-        /// Verifies if the specified file is part of the project.
-        /// </summary>
-        /// <param name="filename">The complete path and filename of the file, relative to the root folder of the project (without leading (back)slash).</param>
-        /// <returns>True if the file is present, otherwise false.</returns>
-        public bool ContainsFile(string filename)
```

Comment:
Is already made available in the parent base class, so no need to implement it here, too.